bcda/service/service_test.go

```diff
@@ -683,53 +704,6 @@ func (s *ServiceTestSuite) TestCancelJob() {
 	}
 }
 
-// TODO: Remove this test once BCDA-4214,4216,4217 are complete.
-// We should be leveraging the time constraints found
-// on the RequestConditions and we shouldn't need this test anymore.
-// Since we do not have any users of it, we need to verify that we set the fields correctly
-func (s *ServiceTestSuite) TestSetTimeConstraints() {
```

Comment:
Conversation starter - table driven tests are awesome, but is there a point that we are making our tests too ambiguous for the sake of ease.  I don't think this is the case here, but are we heading in that direction?  We would have to know to look for TestGetQueJobs if we needed to expand testing on time constraints, or is it that time constraints should always be tested on jobs.